src/renderer/css/components/TabBar.scss

```diff
@@ -0,0 +1,202 @@
+#main {
```

Comment:
Is there a reason to need to add another root div here? Potentially the wrapper .Mainpage should make the styles specific enough. IMO we should specify styled in #root, #app or #main only if they are really global, ex. heading fonts or colours.

Reply:
It's mostly for specificity so that it doesn't interfere with any of the styles provided by the web app.